src/app/core/components/breadcrumbs-2/breadcrumbs-2.component.html

```diff
@@ -0,0 +1,35 @@
+@if (breadcrumbs$ | async; as breadcrumbs) {
+  @if (breadcrumbs.length) {
+    <div class="govuk-breadcrumbs">
+      <ol class="govuk-breadcrumbs__list">
+        @if (!showBackButton) {
+          <li class="govuk-breadcrumbs__list-item">
+            <a class="govuk-breadcrumbs__link" id="breadcrumb-home-link" [routerLink]="['']">Home</a>
+          </li>
+        }
+        @for (breadcrumb of breadcrumbs; track indexOfLink; let last = $last; let indexOfLink = $index) {
+          <li class="govuk-breadcrumbs__list-item">
+            @if (showBackButton) {
+              <a href="#" class="govuk-back-link">Back</a>
```

Comment:
Could this be changed to 
```suggestion
              <a routerLink=".." class="govuk-back-link">Back</a>
```

As this currently does a page reload which gets rid of the feature flags preferences.